tests/main/views/test_direct_award.py

```diff
@@ -1248,10 +1248,18 @@ def test_direct_award_outcome_scenarios(
                         "startDate": None,
                     },
                     "resultOfDirectAward": {
-                        "projectId": project_external_id,
-                        "searchId": active_search_id,
-                        "serviceId": "2000000001",
-                        "archivedServiceId": chosen_archived_service_id,
+                        "project": {
+                            "id": project_external_id,
+                        },
+                        "search": {
+                            "id": active_search_id,
```

Comment:
It's interesting that we went to the effort of making proper public identifiers for projects, but never did it for searches?